src/services/routers/version.py

```diff
@@ -0,0 +1,12 @@
+from fastapi import APIRouter
+from src.core.version import VERSION
+
+router = APIRouter()
+
+
+@router.get("/version", response_model=dict)
+async def get_version():
+    """
+    Returns the current version of the API.
+    """
+    return {"version": VERSION}
```

Comment:
@vasylboliuk please create some models.py module per each router, where you could contain the model you return and define the pydantic model for your responses/requests there. Please rewrite this one to use VersionResponse model

Reply:
Done